File: RolePlayGame/dotnet/fixtures/my.examples/TeenagersFixture.cs
using fit;

using System;
using System.Collections.Generic;

namespace my.examples {

	public class TeenagersFixture : RowFixture {
	
		public override Type GetTargetClass() {
			return typeof(User);
		}
	
		public override object[] Query() {
			UserGateway users = new UserGateway(); // system under test
			List<User> teens = users.Teens();
			return teens.ToArray();
		}
	}

}

File: RolePlayGame/java/fixtures/my/examples/TeenagersFixture.java
package my.examples;

import java.util.List;

import fit.RowFixture;

public class TeenagersFixture extends RowFixture {

	public Class getTargetClass() {
		return User.class;
	}

	public Object[] query() throws Exception {
		UserGateway users = new UserGateway(); // system under test
		List<User> teens = users.teens();
		return teens.toArray();
	}
}
